web/portal/brand/src/entities/Company/Form.tsx

```diff
@@ -39,6 +39,10 @@ const Form = (props: EntityFormProps): JSX.Element | null => {
         continue;
       }
 
+      if (feature.extraData?.iden === ClientTypes.residential) {
+        continue;
+      }
+
       filteredFeatures.push(feature);
     }
```

Comment:
For other types of clients, the displayed options are correct. This block does not include code to perform similar filtering for any other account, so I believe this is not the appropriate place to look.

The API request that populates this dropdown is the same across all client creation forms, so the answer likely lies in the frontend.

Have you checked in Features/SelectOptions?